hackathon/static/hackathon/css/hackathon.css

```diff
@@ -1,7 +1,8 @@
 .hack-score-form {
     width: 100%;
-    margin-right: 0px;
-    margin-left: 0px;
+    margin-right: auto;
```

Comment:
Can't these lines be simplified even more with:
`margin: 0 auto;`
That's the typical method I use mostly.

Reply:
Updated as suggested.